playbook/package.json

```diff
@@ -21,7 +21,7 @@
   "author": "powerhome",
   "license": "ISC",
   "engines": {
-    "node": "10.17.0 || 12.20.0",
+    "node": "10.17.0 || 12.20.1",
```

Comment:
I thought this was one that had been updated upstream earlier by Jasper for GH Actions, but that may not have made it in afterall.